enhancements/vmaas/README.md

```diff
@@ -0,0 +1,472 @@
+---
+title: Virtualization as a service
+authors:
+  - Adrien Gentil
+creation-date: 2025-09-15
+last-updated: 2025-09-15
+tracking-link:
+  - TBD
+see-also:
+replaces:
+superseded-by:
+---
+
+# Virtualization as a service
+
+
+## Summary
+
+We want to provide a service that allows a tenant to create and manage VMs.
+
+## Motivation
+
+TBD
+
+### User Stories
+
+#### As a tenant
+
+- I want to list a pre-defined VM templates
+- I want to create a VM based on a pre-defined template
+- I want a VM that has access to specialized hardware (e.g.: GPU)
+- I want to customize the pre-defined template for my usage
+- I want to attach and detach block storage on my VM that transcends the lifecycle of the VM
+- I want to manage the lifecycle of my VM (start/stop/terminate)
+- I want to import my OS base image
+- I want to list available base OS images
+- I want to manage (create, list, delete) my private network
+- I want to attach my VMs in a virtual private network
+- I want my VM to be exposed outside of my private network
+- I want my VMs to be attached to my own physical L2 network
+- I want to connect on my VM through serial console
+- I want to keep running my VM when a node goes out of service
+
+#### As a service provider:
+
+- I want to define VM templates
+- I want to manually trigger VM migration
```

Comment:
why? What's the "story" of this user story?

Reply:
We wanted to highlight that VM can be re-located seamlessly to another node in case of a scheduled maintenance.